packages/components/psammead-media-player/src/Canonical/index.jsx

```diff
@@ -1,38 +1,63 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import { string } from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+import Guidance from '../Guidance';
+import useTimeout from './useTimeout';
 
-const Canonical = ({ src, title, placeholderSrc }) => {
-  const backgroundStyle = `
-    background-image: url(${placeholderSrc});
-    background-repeat: no-repeat;
-    background-size: contain;
-  `;
+const TIMEOUT_MS = 5000;
 
-  const StyledIframe = styled.iframe`
+const Canonical = ({ service, src, title, placeholderSrc }) => {
+  const [hasTimedOut, setHasTimedOut] = useState(null);
+  const iframeRef = useRef(null);
+
+  useTimeout(setHasTimedOut, iframeRef, TIMEOUT_MS);
+
+  const CanonicalWrapper = styled.div`
+    left: 0;
+    overflow: hidden;
     position: absolute;
     top: 0;
-    left: 0;
     width: 100%;
     height: 100%;
+    ${placeholderSrc &&
+      css`
+        background-image: url(${placeholderSrc});
+        background-repeat: no-repeat;
+        background-size: contain;
+      `}
+  `;
+
+  const StyledIframe = styled.iframe`
     border: 0;
-    overflow: hidden;
-    ${placeholderSrc ? backgroundStyle : ''}
+    height: 100%;
+    width: 100%;
   `;
 
   return (
-    <StyledIframe
-      src={src}
-      title={title}
-      allow="autoplay; fullscreen"
-      scrolling="no"
-      gesture="media"
-      allowFullScreen
-    />
+    <CanonicalWrapper>
+      {hasTimedOut ? (
+        <Guidance
+          service={service}
+          guidanceMessage="The server did not respond. Please check your connection and try again."
```

Comment:
Is this the message dictated by the screen reader in event of a timeout? If so where did the wording come from? Just sounds a bit techie to me 'The server did not respond'.

Reply:
Yes, it will be picked up by screen readers. This feature doesn't actually have a UX specification - it's something they're getting for free. 😄 I will try and rope someone in to take a look. Thanks for reminding me.

Reply:
```
There was a problem loading this content. Please check your internet connection and try again.
```
FYI the above message amendment was contributed by UX this morning, @andrewscfc.